app.js

```diff
@@ -0,0 +1,36 @@
+    {character_name:"Superman",  name:"Clark Kent AKA Kal-el", phone_number: "", comics_affiliation: "DC", image_url: "http://media.dcentertainment.com/sites/default/files/GalleryChar_1900x900_MOS_52e05e3fe24a61.04593858.jpg"},
+    {character_name:"Spiderman",  name:"Peter parker", phone_number: "", comics_affiliation: "Marvel", image_url: "https://cdn0.vox-cdn.com/thumbor/6HMpe8tx99Xv2BYt6Y1Fa0zhrLE=/0x0:1012x675/1280x854/cdn0.vox-cdn.com/uploads/chorus_image/image/45661554/amazing-spider-man-poster-battle-damage.0.0.jpg"},
+    {character_name:"Captain America",  name:"Steve Rogers", phone_number: "", comics_affiliation: "Marvel", image_url: "http://115.imagebam.com/download/IL2fKxg0D0JmWPyleGF3-g/47900/478995554/Captain%20America%201.jpg"},
+    {character_name:"Winter Soldier",  name:"Bucky Barnes", phone_number: "", comics_affiliation: "Marvel", image_url: "http://static.srcdn.com/wp-content/uploads/Sebastian-Stan-Bucky-Barnes-Winter-Soldier-1600x800.jpg"},
+    {character_name:"Wolverine",  name:"James Howlett AKA Logan", phone_number: "", comics_affiliation: "DC", image_url: "http://static.srcdn.com/wp-content/uploads/X.jpg"},
+    {character_name:"Deadpool",  name:"Wade Winston Wilson", phone_number: "", comics_affiliation: "DC", image_url: "http://s3.foxmovies.com/foxmovies/production/films/103/images/gallery/deadpool1-gallery-image.jpg"},
+    {character_name:"Thor",  name:"Thor Odinson", phone_number: "", comics_affiliation: "Marvel", image_url: "http://www.drodd.com/images13/thor14.jpg"},
+    {character_name:"Hulk",  name:"Bruce Banner", phone_number: "", comics_affiliation: "Marvel", image_url: "http://4.bp.blogspot.com/-huySdx74UzY/U24Dcs6PrII/AAAAAAAABRk/WiLSVqQiGWg/s1600/the-hulk+(1).jpg"}
+    // {character_name:"Hulk",  name:"", phone_number: "", comics_affiliation: "", image_url: ""},
+    // {character_name:"Hulk",  name:"", phone_number: "", comics_affiliation: "", image_url: ""},
+    // {character_name:"Hulk",  name:"", phone_number: "", comics_affiliation: "", image_url: ""},
+    // {character_name:"Hulk",  name:"", phone_number: "", comics_affiliation: "", image_url: ""},
+    // {character_name:"Hulk",  name:"", phone_number: "", comics_affiliation: "", image_url: ""},
+    // {character_name:"Hulk",  name:"", phone_number: "", comics_affiliation: "", image_url: ""},
+    // {character_name:"Hulk",  name:"", phone_number: "", comics_affiliation: "", image_url: ""},
+
+]
+
+
+
+
+angular
+.module('superheroesApp', [])
+.controller('superheroesCtrl', [ superheroesController])
+
+ function superheroesController(){
+   this.superheroes = superheroesData;
+   this.addSuperhero = function(){
+   let superheroes = {character_name: this.character_name, name: this.name, phone_number: this.phone_number, comics_affiliation: this.comics_affiliation, image_url: this.image_url}
+   this.superheroes.push(superhero)
```

Comment:
that explains why it was not adding my superheroes. do I still get credit for trying? :)